elephant/statistics.py

```diff
@@ -181,4 +181,56 @@ def fanofactor(spiketrains):
         fano = np.nan
     else:
         fano = spike_counts.var() / spike_counts.mean()
-    return fano
\ No newline at end of file
+    return fano
+
+def lv(v):
+    """
+    Calculate the measure of local variation LV for
+    a sequence of time intervals between events.
+
+    Given a vector v containing a sequence of intervals, the LV is
+    defined as:
+
+    .math $$ LV := \\frac{1}{N}\\sum_{i=1}^{N-1}
+
+                   \\frac{3(isi_i-isi_{i+1})^2}
+                          {(isi_i+isi_{i+1})^2} $$
+
+    The LV is typically computed as a substitute for the classical
+    coefficient of variation for sequences of events which include
+    some (relatively slow) rate fluctuation.  As with the CV, LV=1 for
+    a sequence of intervals generated by a Poisson process.
+
+    Parameters
+    ----------
+
+    v : quantity array, numpy array or list
+        Consecutive time intervals
+
+    Returns
+    -------
+    lvar : float or nan
+           The LV of the inter-spike interval of the input sequence.
+           If an empty list is specified, or if the sequence has less
+           than two one entries, F:=nan.
```

Comment:
I think "two one" is a typo.  And "has less than two entires" includes an empty sequence, so I think the first part is redundant.  So I think it should be "If the sequence has less than two entries, F:=nan"